### ISSUE TITLE
Support additional TypeScript and JavaScript extensions

### ISSUE THREAD

**Issue by tylerbrostrom** [sbarex/SourceCodeSyntaxHighlight#168]
Extension | UTI | Purpose
-- | -- | --
`cjs` | `com.netscape.javascript-source` | Signals to Node.js, “File is a CommonJS module.”
`mts` | `com.microsoft.typescript` | Signals to the TypeScript compiler, “File is an ECMAScript module.”
`cts` | `com.microsoft.typescript` | Signals to the TypeScript compiler, “File is a CommonJS module.”

## Background

afaik, `cjs` has been a valid extension since Node.js 12.

The `mts` and `cts`, however, are brand-new extensions introduced in the [upcoming TypeScript 4.7 release](https://devblogs.microsoft.com/typescript/announcing-typescript-4-7-beta/#esm-nodejs).

Happy to contribute a PR pending approval of the issue :)

**Comment by sbarex** [comment]
The `com.netscape.javascript-source` UTI is already managed by the application. This is the standard UTI for javascript, valid for extensions "js", "jscript", "javascript", "mjs".

What is the output of this Terminal command?
```sh
 mdls -name kMDItemContentTypeTree -name kMDItemContentType DRAG_HERE_A_CJS_FILE
```

**Comment by tylerbrostrom** [comment]
output for a `cjs` file:

```
kMDItemContentType     = "dyn.ah62d4rv4ge80g4xx"
kMDItemContentTypeTree = (
    "public.item",
    "dyn.ah62d4rv4ge80g4xx",
    "public.data"
)
```

**Comment by eshimischi** [comment]
Also curious about adding support for named extensions cause all of em are already a standard

**Comment by sbarex** [comment]
Unfortunately `.mts` format is recognized as a video file (UTI `public.avchd-mpeg-2-transport-stream`).

**Comment by sbarex** [comment]
implemented on release 2.1.12 (61).

**Comment by tylerbrostrom** [comment]
Thanks!
